From 0b9e140ee3e254ac081ec30c5273244a3947180e Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Fri, 20 Jun 2014 18:53:04 +0200
Subject: ARM: shmobile: Move r8a7778.h

Change location of r8a7778.h so it can be included as "r8a7778.h"
instead of the old style <mach/r8a7778.h>

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 02c94f38bf3bcdaaceb1f3486b4dbee1ab550793)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/mach-shmobile/board-bockw-reference.c      | 4 +++-
 arch/arm/mach-shmobile/board-bockw.c                | 4 +++-
 arch/arm/mach-shmobile/{include/mach => }/r8a7778.h | 0
 arch/arm/mach-shmobile/setup-r8a7778.c              | 4 +++-
 4 files changed, 9 insertions(+), 3 deletions(-)
 rename arch/arm/mach-shmobile/{include/mach => }/r8a7778.h (100%)

diff --git a/arch/arm/mach-shmobile/board-bockw-reference.c b/arch/arm/mach-shmobile/board-bockw-reference.c
index 91ff3a27ccc7..ba840cd333b9 100644
--- a/arch/arm/mach-shmobile/board-bockw-reference.c
+++ b/arch/arm/mach-shmobile/board-bockw-reference.c
@@ -19,9 +19,11 @@
  */
 
 #include <linux/of_platform.h>
-#include <mach/r8a7778.h>
+
 #include <asm/mach/arch.h>
+
 #include "common.h"
+#include "r8a7778.h"
 
 /*
  *	see board-bock.c for checking detail of dip-switch
diff --git a/arch/arm/mach-shmobile/board-bockw.c b/arch/arm/mach-shmobile/board-bockw.c
index 8cb057f95c44..b4e4789017bc 100644
--- a/arch/arm/mach-shmobile/board-bockw.c
+++ b/arch/arm/mach-shmobile/board-bockw.c
@@ -34,13 +34,15 @@
 #include <linux/spi/spi.h>
 #include <linux/spi/flash.h>
 #include <linux/usb/renesas_usbhs.h>
+
 #include <media/soc_camera.h>
-#include <mach/r8a7778.h>
 #include <asm/mach/arch.h>
 #include <sound/rcar_snd.h>
 #include <sound/simple_card.h>
+
 #include "common.h"
 #include "irqs.h"
+#include "r8a7778.h"
 
 #define FPGA	0x18200000
 #define IRQ0MR	0x30
diff --git a/arch/arm/mach-shmobile/include/mach/r8a7778.h b/arch/arm/mach-shmobile/r8a7778.h
similarity index 100%
rename from arch/arm/mach-shmobile/include/mach/r8a7778.h
rename to arch/arm/mach-shmobile/r8a7778.h
diff --git a/arch/arm/mach-shmobile/setup-r8a7778.c b/arch/arm/mach-shmobile/setup-r8a7778.c
index b7deec3c3d8d..13512e97cd05 100644
--- a/arch/arm/mach-shmobile/setup-r8a7778.c
+++ b/arch/arm/mach-shmobile/setup-r8a7778.c
@@ -37,11 +37,13 @@
 #include <linux/usb/ehci_pdriver.h>
 #include <linux/usb/ohci_pdriver.h>
 #include <linux/dma-mapping.h>
-#include <mach/r8a7778.h>
+
 #include <asm/mach/arch.h>
 #include <asm/hardware/cache-l2x0.h>
+
 #include "common.h"
 #include "irqs.h"
+#include "r8a7778.h"
 
 /* SCIF */
 #define R8A7778_SCIF(index, baseaddr, irq)			\
-- 
2.1.2

